<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_User</class>
 <widget class="QWidget" name="main_User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_User</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>491</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="accList">
      <property name="accessibleName">
       <string>Profile</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>لیست حساب ها</string>
       </attribute>
       <widget class="QLabel" name="acc_list_lbl">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>90</y>
          <width>123</width>
          <height>80</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>4</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;برای دسترسی به خدمات حساب روی شماره حساب دوبار کلیک کنید&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QListWidget" name="accounts_number_List">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>51</y>
          <width>256</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>66</x>
          <y>30</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;لیست شماره حساب ها&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Profile">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="accessibleName">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;Profile&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <attribute name="title">
        <string>پروفایل</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>19</x>
          <y>19</y>
          <width>281</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="name_lbl">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="birthdate_lbl">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="nc_lbl">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="email_lbl">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>90</y>
          <width>111</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="edit_prof_button">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ویرایش پروفایل</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>تاریخچه</string>
       </attribute>
       <widget class="QListWidget" name="log_listWidget">
        <property name="geometry">
         <rect>
          <x>95</x>
          <y>40</y>
          <width>281</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>166</x>
          <y>20</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;زمان های ورود و خروج&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>ساخت حساب جدید</string>
       </attribute>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>سپرده قرض الحسنه پس انداز</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>سپرده سرمایه گداری کوتاه مدت</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>سپرده سرمایه گداری کوتاه مدت حقوقی</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>سپرده سرمایه گداری بلندمدت</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;نوع حساب&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>80</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;موجودی اولیه&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ad_lineEdit">
        <property name="geometry">
         <rect>
          <x>42</x>
          <y>80</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>150</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>درخواست ساخت حساب</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>خروج</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
